File: Java/AtributosComoObjeto/Autor.java
//Atributos como objeto y método toString

package Java.AtributosComoObjeto;

public class Autor {
    String nombre;
    String nacionalidad;

    public Autor(String nombre, String nacionalidad) {
        this.nombre = nombre;
        this.nacionalidad = nacionalidad;
    }

    @Override
    public String toString(){
        return "Nombre: " + this.nombre + "\n" + "Nacionalidad: " + this.nacionalidad;
    }
}

File: SesionesC#/AtributosComoObjeto/Biblioteca/Autor.cs
public class Autor {
    public string Nombre;
    public string Nacionalidad;

    public Autor(string nombre, string nacionalidad) {
        Nombre = nombre;
        Nacionalidad = nacionalidad;
    }

    public override string ToString() {
        return "Nombre: " + Nombre + "\n" + "Nacionalidad: " + Nacionalidad;
    }
}